.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think you need to add this folder to the gitignore

Reply:
This one was intentional. It let's us share settings as a team, and this setting in particular enables an out-of-the-box experience for the new VS Code feature ([C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit)) that allows developing/building/running at the solution level. This is particularly relevant to this branch - now that there's C# Dev Kit, there's nothing that prevents a Linux or Mac user from installing VS Code and running these script samples directly in the solution. If we wanted to discourage modification of the file, we could add it to our .gitignore. More helpful guidance here, including discussion of suggested default VS code extensions as well, like C# Dev Kit, Python/Jupyter, Matlab syntax highlighting/running):

https://bobbyhadz.com/blog/what-is-vscode-folder#should-you-commit-your-vscode-directory-to-source-control